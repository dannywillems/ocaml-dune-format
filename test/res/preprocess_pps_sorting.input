(library
 (name test)
 (preprocess (pps ppx_deriving.show ppx_jane ppx_base))
 (public_name test)
 (libraries base stdio))