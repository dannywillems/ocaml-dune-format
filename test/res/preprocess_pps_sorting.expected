(library
 (libraries base stdio)
 (name test)
 (preprocess
  (pps ppx_base ppx_deriving.show ppx_jane))
 (public_name test))